cmd/zetaclientd/utils.go

```diff
@@ -199,5 +202,40 @@ func CreateChainObserverMap(
 		}
 	}
 
+	// FIXME_SOLANA: config chain params
+	solChain, solConfig, enabled := appContext.GetSolanaChainAndConfig()
+	solChainParams := observertypes.ChainParams{
+		GatewayAddress: "94U5AHQMKkV5txNJ17QPXWoh474PheGou6cNP2FEuL1d",
+		IsSupported:    true,
+		ChainId:        solChain.ChainId,
+		InboundTicker:  10,
+	}
+
+	// create Solana chain observer if enabled
+	if enabled {
+		rpcClient := solrpc.New(solConfig.Endpoint)
+		if rpcClient == nil {
+			// should never happen
+			return nil, fmt.Errorf("solana create Solana client error")
+		}
+
+		// create Solana chain observer
+		co, err := solanaobserver.NewObserver(
```

Comment:
What does co means? Can we use observer like above?

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a